### PR TITLE
start i18n

### PR DESCRIPTION
**Pull request by groovecoder**
To test ...

1. Download [German Firefox Nightly](https://www.mozilla.org/en-US/firefox/all/?q=German,%20Deutsch#product-desktop-nightly)
2. Run German Firefox (from its .dmg so you don't over-write your English Nightly)
3. Go to `about:debugging`
4. Click "Dieser Nightly"
5. Click "Temporares Add-on laden..."
6. Navigate to this branch's `manifest.json`

Expected results:
The first-run.html page should display in (Google-translated) German!

image

### CODE REVIEW

File: src/_locales/de/messages.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "extensionDescription": {
+        "message": "Firefox Relay makes it easy to create email aliases that forward to your real inbox. Use it to protect your online accounts from hackers and unwanted messages.",
```
Comment (pdehaan) [inline src/_locales/de/messages.json:3]: This isn't German looking.

File: src/_locales/en/messages.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        "description": "Description of the extension. DO NOT TRANSLATE \"Firefox Relay\"."
+    },
+    "firstRunP": {
+        "message": "You’ve successfully installed the Firefox Relay add-on. To get started, sign in to Relay with your Firefox account."
+    },
+    "firstRunCollectDataLabel": {
+        "message": "Allow Mozilla to collect information about how you use this extension."
+    },
+    "disableDataCollection": {
+        "message": "Datensammlung deaktivieren"
```
Comment (pdehaan) [inline src/_locales/en/messages.json:13]: This isn't English looking.

File: src/_locales/de/messages.json (hunk 1 above)
Comment (maxxcrawford) [reply src/_locales/de/messages.json:3]: This is temp file that will be deleted before merging this PR. (We did [the same thing](https://github.com/mozilla/fx-private-relay/pull/815/commits/55ba4d12221ddaa737ba79eb4467a61eba5baf30) when adding it to the main app.)

File: src/_locales/en/messages.json (hunk 2 above)
Comment (groovecoder) [reply src/_locales/en/messages.json:13]: oops, got my copy/pastes crossed. thanks for being thorough!

Comment (groovecoder) [reply src/_locales/en/messages.json]: Fixed this one.

**Comment by groovecoder** [comment]
So, the last message to add contains some dynamic content that could be singular or plural ...

image

Working on this ...

**Comment by pdehaan** [comment]
You have 5/5 remaining aliases
You have 5/5 remaining aliases
You have 5/5 remaining aliases
You have 5/5 remaining aliases

**Review by maxxcrawford — APPROVED** [review]
I found some additional strings not translated yet. Besides that, this looks great. I'm approving this for a fast-follow up. 

- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/add_input_icon.js#L176
- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/add_input_icon.js#L200
- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/add_input_icon.js#L211
- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/add_input_icon.js#L222
- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/add_input_icon.js#L263

image

**Comment by maxxcrawford** [comment]
Correction – I found a few more: 

- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/fill_relay_address.js#L45
- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/fill_relay_address.js#L31
- https://github.com/mozilla/fx-private-relay-add-on/blob/i18n/src/js/fill_relay_address.js#L26

**Comment by groovecoder** [comment]
I think I got all the remaining English content. Revealed a bit of on-boarding panel UI that may need some more margin or padding to adjust for more verbose languages?

image

**Comment by groovecoder** [comment]
Merging this to un-block future merges.

**Pull request merged**